tests/Projection/MySQLEventStoreProjectionTest.php

```diff
@@ -12,216 +12,33 @@
-
-        $projection
-            ->fromStream('user-123')
-            ->when([
-                UserCreated::class => function (array $state, UserCreated $event): array {
-                    $this->emit($event);
-                    $this->stop();
-
-                    return $state;
-                },
-            ])
-            ->run();
-
-        $streams = $this->eventStore->load(new StreamName('test_projection'));
-        $events = $streams->streamEvents();
-
-        $this->assertCount(1, $events);
-        $this->assertEquals('Alex', $events->current()->payload()['name']);
-
-        $projection->delete(true);
-
-        $this->expectException(StreamNotFound::class);
-        $this->eventStore->load(new StreamName('test_projection'));
-    }
-
-    /**
-     * @test
-     */
-    public function it_ignores_error_on_delete_of_not_created_stream_projections(): void
-    {
-        $this->prepareEventStream('user-123');
-
-        $projection = $this->eventStore->createProjection('test_projection');
-
-        $projection
-            ->fromStream('user-123')
-            ->when([
-                UserCreated::class => function (array $state, UserCreated $event): array {
-                    $this->stop();
-
-                    return $state;
-                },
-            ])
-            ->run();
-
-        $projection->delete(true);
-    }
-
-    /**
-     * @test
-     */
-    public function it_throws_exception_on_run_when_nothing_configured(): void
-    {
-        $this->expectException(RuntimeException::class);
-
-        $projection = $this->eventStore->createProjection('test_projection');
-
-        $projection->run();
-    }
-
-    /**
-     * @test
-     */
-    public function it_throws_exception_when_trying_to_run_two_projections_at_the_same_time(): void
-    {
-        $this->expectException(\Prooph\EventStore\PDO\Exception\RuntimeException::class);
-        $this->expectExceptionMessage('Another projection process is already running');
-
-        $this->prepareEventStream('user-123');
-
-        $projection = $this->eventStore->createProjection('test_projection');
-
-        $eventStore = $this->eventStore;
-
-        $projection
-            ->fromStream('user-123')
-            ->whenAny(
-                function (array $state, Message $event) use ($eventStore): array {
-                    $projection = $eventStore->createProjection('test_projection');
 
-                    $projection
-                        ->fromStream('user-123')
-                        ->whenAny(
-                            function (array $state, Message $event): array {
-                                $this->linkTo('foo', $event);
+        $this->connection = TestUtil::getConnection();
+        $this->connection->exec(file_get_contents(__DIR__.'/../../scripts/mysql/01_event_streams_table.sql'));
+        $this->connection->exec(file_get_contents(__DIR__.'/../../scripts/mysql/02_projections_table.sql'));
```

Comment:
This lines could be added in TestUtil as "initDefaultDatabaseTables" method, since they are used on other test parts too.